package.json

```diff
@@ -40,14 +40,14 @@
   },
   "devDependencies": {
     "dotenv-cli": "^7.3.0",
-    "eslint": "^8.54.0",
-    "prettier": "^3.1.0",
-    "prettier-plugin-tailwindcss": "^0.5.7",
-    "ts-node": "^10.9.1",
+    "eslint": "^8.56.0",
+    "prettier": "^3.2.4",
+    "prettier-plugin-tailwindcss": "^0.5.11",
+    "ts-node": "^10.9.2",
     "tsc-watch": "^6.0.4",
-    "turbo": "^1.10.16",
-    "turbo-ignore": "^1.10.16",
-    "typescript": "^5.2.2",
+    "turbo": "^1.11.3",
+    "turbo-ignore": "^1.11.3",
+    "typescript": "^5.3.3",
```

Comment:
Did the dependencies get bumped automatically or something here?

Reply:
eghrhrhm someone updated trpc in some "modules" (or what they are called) but not in this one so i had to update it, but i can udno everything other than trpc